### PR TITLE
Add contribution guidelines

### LINKED ISSUE #9: [DOCS] Add contribution guidelines
### Description

I can work on this if required.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by thevinitgupta**
## 🛠️ Fixes Issue

Close #9 

## 👨‍💻 Changes proposed

Add contribution guidelines

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+> IMPORTANT **Note**
+> **Pull Requests having no issue associated with them will not be accepted. Firstly get an issue assigned, whether it's already opened or raised by you, and then create a Pull Request.**
```
Comment (Pradumnasaraf) [inline CONTRIBUTING.md:2]:
```suggestion
> **Pull Requests having no issue associated with them will not be accepted. Firstly get an issue assigned, whether it's already opened or raised by you, and then create a Pull Request.**

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Just a blank line need to added

**Comment by thevinitgupta** [comment]
I have made the required changes

**Comment by Pradumnasaraf** [comment]
Hey,  I have requested to add a blank like after line 3. You can check comment here https://github.com/Pradumnasaraf/Botchain/pull/25#discussion_r1111234456

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge :tada:.

**Pull request merged**